tests/features/bootstrap/SyncContext.php

```diff
@@ -10,51 +18,91 @@
 
 class SyncContext extends BehatContext {
 
+  // Protected variables for the tests to use
+  protected static $queueID;
   protected static $categorySitemap;
   protected static $totalStatus;
 
   /**
-   * @Given /^I sync (\d+) most recent posts from the category sitemap "([^"]*)"$/
+   * @Given /^I purge the queue$/
+   */
+  public function iPurgeTheQueue()
```

Comment:
Pop the opening curly brace on this line, ta. Check the other functions below too.